### PR TITLE
asadiqbal08/mitxpro-149 Find submission history using learner's email address

### PR DESCRIPTION
**Pull request by asadiqbal08**
Task: #149 
Can't find submission history using learner's email address

### CODE REVIEW

**Review by ahmed-arbisoft — APPROVED** [review]
LGTM :shipit:

**Comment by asadiqbal08** [comment]
@pdpinch any thought over it else merging it.

File: lms/djangoapps/courseware/views/views.py [hunk 1]
```diff
@@ -1171,12 +1171,19 @@ def _course_home_redirect_enabled():
...
     """Render an HTML fragment (meant for inclusion elsewhere) that renders a
     history of all state changes made by this user for this problem location.
     Right now this only works for problems because that's all
     StudentModuleHistory records.
     """
+    try:
+        if '@' in student_username_or_email:
+            user = User.objects.get(email=student_username_or_email)
+        else:
+            user = User.objects.get(username=student_username_or_email)
```
Comment (pdpinch) [inline lms/djangoapps/courseware/views/views.py]: Touchstone collaborator accounts will have `@` in the username, which is allowed in edX when unicode usernames are turned on in settings.

You might want to look at how we handled this in similar situations, like https://github.com/mitodl/edx-platform/pull/70

Comment (asadiqbal08) [reply lms/djangoapps/courseware/views/views.py]: @pdpinch done. take another look.

**Review by annagav — APPROVED** [review]
👍

**Pull request merged**